jax/_src/api.py

```diff
@@ -1750,6 +1754,95 @@ def pmap(
       global_arg_shapes=global_arg_shapes)
 
 
+class PmapCallInfo(NamedTuple):
+  flat_fun: lu.WrappedFun
+  in_tree: PyTreeDef
+  out_tree: PyTreeDef
+  flat_args: Iterable[Any]
+  donated_invars: Iterable[bool]
+  in_axes_flat: Any
```

Comment:
Don't we have a better annotation for this? I would expect it's a `Sequence[Optional[int]]`